phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1420,7 +1420,7 @@ public MetaDataMutationResult createTable(final List<Mutation> tableMetaData, fi
         byte[] tenantIdBytes = rowKeyMetadata[PhoenixDatabaseMetaData.TENANT_ID_INDEX];
         byte[] schemaBytes = rowKeyMetadata[PhoenixDatabaseMetaData.SCHEMA_NAME_INDEX];
         byte[] tableBytes = rowKeyMetadata[PhoenixDatabaseMetaData.TABLE_NAME_INDEX];
-        byte[] tableName = physicalTableName != null ? physicalTableName : SchemaUtil.getTableNameAsBytes(schemaBytes, tableBytes);
+        byte[] tableName = SchemaUtil.getTableNameAsBytes(schemaBytes, tableBytes);
```

Comment:
Why we need this change? I think it should be physical table name, we need it to verify meta data when views created on phoenix table (create view v as select * from table) in ensureTableCreated(tableName, tableType, tableProps, families, splits, true, isNamespaceMapped);